### DISCUSSION TITLE
9键状态下，为什么只能发英文？

### DISCUSSION THREAD

**Discussion by RunYou** [imfuxiao/Hamster#365] [Q&A]
![image](https://github.com/imfuxiao/Hamster/assets/43163474/28c024c7-ed6b-4ed8-883f-eec67c08809a)
今天刚从1.0升级到2.0。如图，26键状态下就是正常的拼音输入

同时这个键盘字母太大，能否调小一点？

**Comment by imfuxiao** [comment]
中文九宫格需要搭配九宫格的输入方案，目前雾凇已支持，建议导入雾凇后选择“雾凇·九宫格”方案后搭配使用。

**Reply by RunYou** [reply]
雾凇九宫格在哪?我1.0使用的雾凇方案，目前从1.0刚刚升级到2.0，在方案中并没有看到雾凇九宫格选项，请问需要重新再导入一次雾凇么?
感谢开发者回答

**Reply by imfuxiao** [reply]
你的雾凇方案应该需要更新下，雾凇方案也是在2.0后同步更新并添加了仓的支持。

**Reply by RunYou** [reply]
> 你的雾凇方案应该需要更新下，雾凇方案也是在2.0后同步更新并添加了仓的支持。

了解，非常感谢您的解答。还有一个问题，想改一下九宫格输入法皮肤的字体大小，就是按键上的字母大小，请问应该改哪个文件?我刚刚翻了一圈没找到，所以来问一下，谢谢

**Reply by imfuxiao** [reply]
目前还不能调整键盘上按键显示的字体大小，不过后面有开发计划支持。
